pages/Identity/Registry.md

```diff
@@ -6,21 +6,228 @@ menu: Identity
 
 # Identity Registry
 
+A registry allows for distinct user identifiers and human readable discovery of network members.
+The DSNP Id Registry is a simple contract that allows switching handles, identity contracts, and chain migration
```

Comment:
(Goes down a bit of a rabbit trail of Id vs ID. Hehehe or it could be the DSNP id or DSNP super ego)

I was thinking of it as an abbreviation of `DSNP Identifier` instead of `DSNP Identity Document`, but I don't think I have a strong preference ether way. When we use it as a variable in the interface we just use `id`, so no real guidance there. In other code I would expect either `dsnpId` or `dsnpID` to be fine, although likely the first in case additional words come after it (for readability: `dsnpIdForSending` is easier to read imho than `dsnpIDForSending` although either would be better with a serif on that capital I.)

Reply:
I like "Id" a little better, but it's not a strong opinion. We could just side-step the issue here by writing out "Identifier."

Reply:
>I was thinking of it as an abbreviation of DSNP Identifier instead of DSNP Identity Document
Yea that makes sense.

TBH (or Tbh? if you prefer) I like dsnpAndThenTheCamelCase but ¯\_(ツ)_/¯